<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BuscarR</class>
 <widget class="QWidget" name="BuscarR">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>737</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTextEdit" name="txtCodTT">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>90</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HP Simplified Light</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QTextEdit" name="txtCodT">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>130</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HP Simplified Light</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>180</y>
     <width>270</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HP Simplified Light</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ingrese el pais de salida: </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_23">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>220</y>
     <width>331</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HP Simplified Light</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ingrese la ciudad de salida: </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>140</y>
     <width>270</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HP Simplified Light</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ingrese el código del tren: </string>
   </property>
  </widget>
  <widget class="QTextEdit" name="txtCodCS">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>210</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HP Simplified Light</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_24">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>30</y>
     <width>171</width>
     <height>36</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HP Simplified Light</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>BUSCAR RUTA</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>100</y>
     <width>270</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HP Simplified Light</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ingrese el código del tipo de tren: </string>
   </property>
  </widget>
  <widget class="QTextEdit" name="txtCodPS">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>170</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HP Simplified Light</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="btCancelarPrecio_5">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>430</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HP Simplified Light</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Consultar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btCancelarTiempo">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>430</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HP Simplified Light</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Salir</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_25">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>300</y>
     <width>331</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HP Simplified Light</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ingrese la ciudad de llegada: </string>
   </property>
  </widget>
  <widget class="QTextEdit" name="txtCodCL">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>290</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HP Simplified Light</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_22">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>260</y>
     <width>270</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HP Simplified Light</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ingrese el pais de llegada: </string>
   </property>
  </widget>
  <widget class="QTextEdit" name="txtCodPL">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>250</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HP Simplified Light</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
